src/components/TagPicker/index.js

```diff
@@ -37,25 +37,30 @@ function TagPicker({selectedTag, tag, policyTags, policyRecentlyUsedTags, onSubm
         ];
     }, [selectedTag]);
 
+    const enabledTags = useMemo(() => {
+        const sortedTagList = _.chain(policyTagList)
+            .values()
+            .sortBy((policyTag) => policyTag.name)
+            .value();
+
+        if (!shouldShowDisabledAndSelectedOption) {
+            return sortedTagList;
+        }
```

Comment:
Actually I don't see the need of this early return.
Can't we apply same logic to both new request and edit request?

Reply:
I'm not too sure either. The name suggests that we display disabled tags, but I'm not sure that that's happening

Reply:
@tienifr @robertKozik could you check this logic? It seems like we'd want to return the `policyTagList` if `shouldShowDisabledAndSelectedOption` is true and not the inverse like we have in the code, right?